x-pack/elastic-agent/pkg/agent/operation/operation_config.go

```diff
@@ -7,6 +7,8 @@ package operation
 import (
 	"context"
 
+	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/core/plugin/state"
```

Comment:
mage fmt should keep things together

Reply:
I ran `mage fmt`, it passed lint as well, so it believe this is valid. But I will fix.